website/src/remark/render-code-blocks.mjs

```diff
@@ -171,12 +171,18 @@ export default function plugin() {
             const skipErr = meta && meta.match(skipRegex) !== null;
             const editableRegex = new RegExp(/(always-editable)/g);
             const alwaysEditable = meta && meta.match(editableRegex) !== null;
+            const lineNumRegex = new RegExp(/(showLineNumbers)/g);
```

Comment:
or
    const lineNumRegex = /(showLineNumbers)/g

Reply:
maybe `/ig` to be case insensitive

Reply:
resolved in #98